doc/user_guide/user_guide.md

```diff
@@ -58,7 +58,23 @@ To install the administration scripts, run the SQL batch file `administration-sq
 
 ### Role-based security
 
-Role-based security is a way to secure a table by assigning one or more roles to each user and specifying the roles which are allowed to see a row for each row in the table. 
+Role-based security is a way to secure a table by assigning one or more roles to each user and specifying the roles which are allowed to see a row for each row in the table.
+
+#### Roles
+
+A role in the real world is a responsibility that comes with certain privileges. In a soccer team for example you have the role of a goalkeeper, defenders or a coach. All have different responsibilities and different things they are allowed to do. The goalkeeper for example is the only player in the game allowed to touch the ball with the hands.
+
+Don't confuse roles with groups. In our example the football team would be a group. We will talk about [groups in a later section](#group-based-security).
+
+RLS supports up to 63 general-purpose roles. Those can be freely named by you.
+
+You assign roles to users and data rows and RLS matches the assigned roles to determine if a user is allowed to access a row or not.
+
+There is one additional reserved role that always exists. The *public role*. If you assign this role to a row in RLS, every user with access to the schema can see this row, independently of that user's own roles. So even if a user has no roles assigned at all, that user can still see all rows that have the public role set.
```

Comment:
The guide never says how to assign the public role in this section: https://github.com/exasol/row-level-security/blob/main/doc/user_guide/user_guide.md#protecting-a-table-with-role-based-rls
Somehow the bit `BIT_SET(0, 63)` has to be set, but is there an interface like `ROLES_MASK()` for this?

Also, maybe mention which ID the public role has and that it has no name because it's not in `EXA_ROLES_MAPPING` and thus also doesn't occur in any of the list function's outputs.

Reply:
You are right. I wrote a feature ticket for that and will do that in the next feature release.
https://github.com/exasol/row-level-security/issues/96